app/templates/components/modals/event-delete-modal.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="header">
+  {{t 'Are you sure you would like to delete this event?'}}
+  <div class="muted small text">
+    {{t 'Deleting the event will delete all the data associated with it.'}}
+  </div>
+</div>
+<div class="content">
+  <form class="ui form" autocomplete="off" {{action deleteEvent on='submit' preventDefault=true}}>
+    <div class="field">
+      <div class="label">
+        {{t 'Please enter the full name of the event to continue'}}
+      </div>
+      {{input type='text' name='confirm_name' value=confirmName}}
+    </div>
+  </form>
+</div>
+<div class="actions">
+  <button type="button" class="ui black button" {{action 'close'}}>
+    {{t 'Cancel'}}
+  </button>
+  <button type="submit" class="ui red button" disabled={{doesNameMatch}}  {{action deleteEvent}}>
```

Comment:
Oh, okay then I misunderstood the error log, updating the test. And what about the